calculator_test.go

```diff
@@ -5,20 +5,82 @@ import (
 	"testing"
 )
 
-func TestAdd(t *testing.T) {
-	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
-	if want != got {
-		t.Errorf("want %f, got %f", want, got)
+func TestCalculator(t *testing.T) {
+
+	var tests = []struct {
+		function func(float64, float64) float64
+		want     float64
+		input    []float64
```

Comment:
We might name these `a` and `b`, if there are just two of them; on the other hand, if you're doing the stretch goal of extending `Multiply` and friends to handle any number of parameters, keep it as a slice!